demo/text-field-custom-demos.html

```diff
@@ -0,0 +1,78 @@
+<dom-module id="text-field-custom-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <h3>Custom Input</h3>
+    <p>
+      <b>Note:</b> In order to apply lumo or material theme styles to the placeholder
+      of the slotted input you should explicitly import a file
+      `theme/lumo/vaadin-placeholder-styles.html` or `theme/material/vaadin-placeholder-styles.html`
```

Comment:
Better not use backticks in the demo descriptions as they are not parsed as Markdown.

Reply:
I still see lots of backticks in the demos

Reply:
Ping @yuriy-fix